### ISSUE TITLE
PF4 Modal component is not draggable/movable

### ISSUE THREAD

**Issue by hstastna** [patternfly/patternfly#3789]
We are missing the draggable/movable ability regarding the PF4 React `Modal` component. This is more like a feature request. 

Sometimes it is very unpleasant not to see the important info under the dialog window. Having it draggable would solve such problems in any UI.

This could be relatively easily solved by using the PF3 component, but the effort is to go forward and to use only the PF4 components. Moving back and supporting PF3 is not a way to go, together with some other solutions we considered in my team.

This is blocking the BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1744114

Thanks very much in advance for any info, comments, suggestions!

**Comment by hstastna** [comment]
I think this issue is still actual. WDYT, @mcarrano? Thanks! :)

**Comment by mcarrano** [comment]
Yes, we are planning to begin work on this is our upcoming sprint.

**Comment by mcoker** [comment]
@mcarrano per our discussion to begin this in react, want to move this issue or open a new issues in the react repo to track that work?

**Comment by mcarrano** [comment]
Yes, I will transfer to React
